docs/references/message-execution-properties.mdx

```diff
@@ -14,26 +14,26 @@ The [community conversation on security best practices](https://www.youtube.com/
 
 To interact with a canister's methods, there are two primary types of **calls** that can be used: [update](/references/ic-interface-spec.md#http-call) calls and [query](/docs/current/references/ic-interface-spec#http-query) calls.
```

Comment:
Not part of your change, but apart from mentioning updates and queries here and in the paragraph below, we never clarify anything about either of them.

I haven't followed the link, but by the looks of the URL (involving HTTP endpoints), it refers to "replicated updates" and "non-replicated queries". Which is even more confusing, as further down we say "Each downstream call that a canister makes, query or update, triggers a message." Which may imply that a replicated message execution can somehow make a non-replicated query call.

Maybe it's necessary to explicitly state that this page is all about replicated execution (which I guess it is, but I'm not 100% sure, because there's also composite queries, which behave somewhat similarly to replicated execution). And briefly explain the difference between replicated updates and replicated queries.

Reply:
I've removed all the details about HTTP calls and focused on the endpoints - not sure if you're somehow looking at an old version of the PR...